content/docs/2.0/concepts/external-scalers.md

```diff
@@ -0,0 +1,582 @@
++++
+title = "External Scalers"
+weight = 500
++++
+
+While KEDA ships with a set of [built-in scalers](../scalers), users can also extend KEDA through a GRPC service that implements the same interface as the built-in scalers.
```

Comment:
Maybe we can extend this bit with the different external scalers and how they are different before we dive in the details?

Reply:
I added couple more sentences. Does that match what you were thinking?

There isn't much difference between built-in and external scalers other than the responsibility of managing the GRPC server yourself.